sphinx_automodapi/automodsumm.py

```diff
@@ -313,6 +313,13 @@ def automodsumm_to_autosummary_lines(fn, app):
 
     """
 
+    if SPHINX_LT_16:
+        warn = app.warn
```

Comment:
so this has to be done for all the functions? Any way to reuse L438? (I suppose there isn't, and anyway, we should really just drop <1.6).

Reply:
Yeah I think I'd rather just drop <1.6 - see https://github.com/astropy/sphinx-automodapi/issues/85 in case you have any opinions.

Reply:
indeed, apparently I forgot to comment there 😅